sw/device/tests/power_virus_systemtest.c

```diff
@@ -857,6 +869,23 @@ static void configure_otbn(void) {
                                              &rsa3072_constants));
 }
 
+static void check_crypto_blocks_idle(void) {
```

Comment:
Another thing to consider is to gate the clocks of crypto cores using [CLK_HINTS register of the clock manager](https://docs.opentitan.org/hw/ip/clkmgr/doc/#Reg_clk_hints). This can be done in another PR, just wanted to bring this to your attention :)

Reply:
Yes good idea. I made note [here](https://github.com/lowRISC/opentitan/issues/14814#issuecomment-1451372465).